kotlin/src/test/kotlin/Test.kt

```diff
@@ -37,9 +37,9 @@ internal class LaneTest {
             if (!testCase.skip) {
                 val parsed = Road(testCase.tags, testCase.driving_side).parse()
                 assertEquals(
-                    testCase.output,
+                    testCase.output.filter { it.type != LaneType.SEPARATOR },
                     parsed,
-                    "\nGot:      " + parsed + "\nExpected: " + testCase.output + "\n"
+                    "${testCase.way_id}\nGot:      ${parsed}\nExpected: ${testCase.output}\n"
```

Comment:
The Kotlin code is still using the JSON, but now that file is deleted from git and under `.gitignore`. Is this just temporary? The goal is to move everything to using YAML?

Reply:
I couldn't get native yaml parsing working, so I have the python dump a json :)
feel free to fix it, but im not good at the kotlin + the DSL for dependencies :P